ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -96,7 +96,7 @@ Item {
                         messageContextMenu.isProfile = true
                         messageContextMenu.myPublicKey = userProfile.pubKey
                         messageContextMenu.selectedUserPublicKey = model.pubKey
-                        messageContextMenu.selectedUserDisplayName = model.displayName
+                        messageContextMenu.selectedUserDisplayName = nickName || userName
```

Comment:
Might be safer to update in the model. Looks like there are more checks involved in choosing the right display name.

Reply:
Well this is a different model and it has to display both (nickname and displayName); corrected to use the list item's `title` as that contains the required logic already